lte/gateway/python/magma/monitord/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+LTE_ROOT = "../../"
+
+ORC8R_ROOT = LTE_ROOT + "../../../orc8r/gateway/python"
+
+py_binary(
+    name = "monitord",
+    srcs = ["main.py"],
+    imports = [
+        LTE_ROOT,
+        ORC8R_ROOT,
+    ],
+    # legacy_creat_init = False is required to fix issues in module import, see https://github.com/rules-proto-grpc/rules_proto_grpc/issues/145
+    legacy_create_init = False,
+    main = "main.py",
+    visibility = ["//visibility:private"],
+    deps = [
+        ":monitord_lib",
+        "//lte/protos:mconfigs_python_proto",
+        "//lte/protos:mobilityd_python_proto",
+        "//orc8r/gateway/python/magma/common:rpc_utils",
+        "//orc8r/gateway/python/magma/common:sentry",
+        "//orc8r/gateway/python/magma/common:service",
+    ],
+)
+
+py_library(
+    name = "monitord_lib",
+    srcs = [
+        "cpe_monitoring.py",
+        "icmp_job.py",
+        "icmp_state.py",
+    ],
+    visibility = ["//visibility:public"],
```

Comment:
Question (again, perhaps for @nstng): I recall asking this question previously but don't know how it settled.  In my time at Google we typically embedded visibility information in the package declaration at top of BUILD.bazel (rarely placing it in individual rules).  I get that this is less granular and - if e.g. package is public - could lead to default behavior that is not what is intended. But it seemed generally fine (and reduces a lot of per-target boilerplate). What are your thoughts?

Reply:
We already have a couple of cases where some libraries are private, e.g., we started to use the pattern to have a `py_binary` only reference the `main.py` and add other source files as a `py_library` (this greatly improves the readability of dependencies). If the `py_library` is not tested then there is currently no reason to make it public.
Also I was thinking about to introduce grouped visibility - i.e., a target can only be used by a certain subset of packages.
In my opinion a strict visibility policy can mitigate uncontrolled growth over time. But if you prefer to reduce the boilerplate then we can change this - my opinion is not that strong on this topic.

Reply:
I don't presently have strong opinions either. On a project like Magma I would generally assume that libraries can be used anywhere in the repo (we are not trying to limit dependencies by other teams). But I agree that some files make no sense to be used by others (e.g. simple main.py files).  For test purposes, I often see use of `testonly = 1,` to prevent use in production code (better than visibility for this purpose).

Reply:
We will discuss this again in the team. And we will have a look at and start using `testonly = 1` (did not had this on my radar - thank you)

Reply:
Added a revise task to our backlog.